src/ui/issuepanel/IssuePanel.java

```diff
@@ -38,6 +40,11 @@
 	private final KeyCombination defaultSizeWindow = new KeyCodeCombination(KeyCode.D, KeyCombination.CONTROL_DOWN);
 	private HashMap<Integer, Integer> issueCommentCounts = new HashMap<>();
 
+	private static final int BROWSER_REQUEST_DELAY = 500; //milliseconds
+	private static TickingTimer timer;
```

Comment:
The timer shouldn't be static. If there is conceptually only one timer for all panels, it would fit better in ColumnControl.